README.md

```diff
@@ -12,4 +12,4 @@ data: MyData = ImplicitDict.parse({'foo': 'asdf', 'bar': 1}, MyData)
 assert json.dumps(data) == '{"foo": "asdf", "bar": 1}'
 ```
 
-See [class documentation for `ImplicitDict`](src/implicitdict/__init__.py) and [test_normal_usage.py](tests/test_normal_usage.py) for more information.
+See [class documentation for `ImplicitDict`](https://github.com/interuss/implicitdict/src/implicitdict/__init__.py) and [test_normal_usage.py](https://github.com/interuss/implicitdict/tests/test_normal_usage.py) for more information.
```

Comment:
The links are incorrect leading to 404. The correct version should reference the branch. ie https://github.com/interuss/implicitdict/blob/main/src/implicitdict/__init__.py

Reply:
Thanks for that catch